pkg/preflight/assets/host-preflights.yaml

```diff
@@ -536,13 +536,6 @@ spec:
           - pass:
               when: "timezone == UTC"
               message: "Timezone is set to UTC"
-    - hostOS:
-        checkName: "Collectd Support"
-        exclude: '{{kurl not .Installer.Spec.Collectd.Version}}'
-        outcomes:
-          - fail:
-              when: "ubuntu = 22.04"
-              message: "Collectd addon not supported on ubuntu 22.04"
```

Comment:
@emosbaugh can we move with this one?

See that I am adding a new test spec to the daily Testgrids because we are not running test the all metrics AddOns added with kots. Either we have no tests so far with collected so I think would be beneficial we cover its install.